src/cascadia/TerminalApp/ColorScheme.h

```diff
@@ -29,35 +29,76 @@ namespace TerminalAppLocalTests
 
 namespace winrt::TerminalApp::implementation
 {
+// Use this macro to quick implement both the getter and setter for a color property.
+// This should only be used for color types where there's no logic in the
+// getter/setter beyond just accessing/updating the value.
+// This takes advantage of til::color
+#define GETSET_COLORTABLEPROPERTY(name)                                               \
+public:                                                                               \
+    winrt::Windows::UI::Color name() const noexcept { return _table[##name##Index]; } \
+    void name(const winrt::Windows::UI::Color& value) noexcept { _table[##name##Index] = value; }

```

Comment:
do we need an observable/notifier here?

Reply:
Maybe? I'll hold off on that for when we get the UI hooked up.